plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/RenderJob.java

```diff
@@ -328,12 +364,20 @@ public boolean isConflicting(ISchedulingRule rule) {
                 return true;
             else if (getRenderer() instanceof CompositeRendererImpl)
                 return false;
-            else
-                return other.getRenderer().getClass().equals(getRenderer().getClass())
-                        && !isCanceled() && !other.isCanceled()
-                        && isUsingLocalResources(getRenderer())
-                        && isUsingLocalResources(other.getRenderer());
 
+            if (other.getRenderer().getClass().equals(getRenderer().getClass()) && !isCanceled()
+                    && !other.isCanceled()) {
+
+                if (isUsingLocalResources(getRenderer())
+                        && isUsingLocalResources(other.getRenderer())) {
+                    return true;
+                }
+                if (isConflictingGeoResources(other.getRenderer())) {
+                    return true;
+                }
+            }
```

Comment:
Please correct me if I'm wrong. You added another check `isConflictingGeoResources`  ..

Reply:
I will revert the addition of `isConflictingGeoResources()`